KachnaOnline.Business/Services/UserService.cs

```diff
@@ -13,15 +13,19 @@
 using KachnaOnline.Business.Configuration;
 using KachnaOnline.Business.Constants;
 using KachnaOnline.Business.Data.Repositories.Abstractions;
+using KachnaOnline.Business.Exceptions.Roles;
 using KachnaOnline.Business.Models.Kis;
 using KachnaOnline.Business.Models.Users;
 using KachnaOnline.Business.Services.Abstractions;
-using KachnaOnline.Data.Entities.Users;
+using Microsoft.EntityFrameworkCore;
```

Comment:
Why?

Reply:
It's required when getting all users/roles in order to use `ToListAsync()`. If you are asking why `KachnaOnline.Data.Entities.Users` was removed, it's no longer necessary. There were name clashes between the models and entities so I decided to go with the solution that you can see `using UserEntity = KachnaOnline.Data.Entities.Users.User;` etc